### ISSUE TITLE
[Suggestion] Add support for publishing messages

### ISSUE THREAD

**Issue by GMatrixGames** [DV8FromTheWorld/JDA#1357]
## General Troubleshooting
  
- [x] I have checked for similar issues.
- [x] I have updated to the [latest JDA version][download].
- [x] I have checked the branches or the maintainers' PRs for upcoming features.

## Feature Request
I'd like to suggest the integration of publishing messages since the opening of news channels to every discord. And messages from bots don't auto-publish

### Example Use-Case
A use case would be if a server has a bot that posts a tweet in a news channel every time a specific account tweets. People wouldn't get the message if they follow that channel since it doesn't publish on post.

**Comment by RedstoneDaedalus** [comment]
It's still entirely possible Discord will block this endpoint for bots, it is better to wait for them to add it to the developer documentation firwt.

**Comment by GMatrixGames** [comment]
I mean, discord.py added it already, so I'm fairly certain it's fine.

**Comment by GMatrixGames** [comment]
@MinnDevelopment If you don't mind me asking, what does the freezer status label mean?

**Comment by averen** [comment]
> It's still entirely possible Discord will block this endpoint for bots, it is better to wait for them to add it to the developer documentation first.

As per discord/discord-api-docs#1692 it is quite clear they are enthusiastic about exposing these endpoints. The freezer label are for issues that are waiting on another issue to be resolved, in this case it is waiting for the documentation to be merged. Until then, implementation details may change as they clean up the feature on their end.

**Comment by GMatrixGames** [comment]
Ah alright. I'll be awaiting the PR to get pulled on discord's end.

**Comment by Andre601** [comment]
> @MinnDevelopment If you don't mind me asking, what does the freezer status label mean?

You can just hover over the label to get a tooltip with its description:
`this is currently put on hold`, so essentially freezer means, the PR/issue won't be worked on for reasons.

**Comment by GMatrixGames** [comment]
> You can just hover over the label to get a tooltip with its description:
> this is currently put on hold, so essentially freezer means, the PR/issue won't be worked on for reasons.

I was on mobile as it was 11PM, and that feature doesn't exist on there. Otherwise I wouldn't have asked.
